x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -213,13 +214,12 @@ func runCmd(
 
 		var cmdError *SynthError = nil
 		if err != nil {
-			errMessage := fmt.Sprintf("command exited with status %d: %s", cmd.ProcessState.ExitCode(), err)
-			cmdError = &SynthError{Name: "cmdexit", Message: errMessage}
+			cmdError = ECSErrToSynthError(ecserr.NewBadCmdStatusErr(cmd.ProcessState.ExitCode(), loggableCmd.String()))
```

Comment:
The original error message is hidden here, Is this intended?

Reply:
Yes! That error is just the string "exited with status 123", so it adds nothing.